README.md

```diff
@@ -29,3 +29,41 @@ Then do the following with your openshift/origin pull request:
 1. Drop the TMP commit (pointing glide back at openshift/api#master)
 2. Rerun `hack/update-deps.sh` and update your `bump(*)` commit
 3. It can then be tagged and merged by CI
+
+## generating CRD schemas
+
+Since Kubernetes 1.16, every CRD created in `apiextensions.k8s.io/v1` is required to have a [structural OpenAPIV3 schema](https://kubernetes.io/blog/2019/06/20/crd-structural-schema/). The schemas provide server-side validation for fields, as well as providing the descriptions for `oc explain`. Structural requirements build on that to provide enhanced security in objects' persistence in etcd. As we host many of our CRDs in this repo along with their corresponding Go types we also require them to have schemas. However, the following instructions apply for CRDs that are not hosted here as well.
```

Comment:
> Structural requirements build on that to provide enhanced security in objects' persistence in etcd

Not sure I can follow. You mean: "Moreover, schemas ensure structural consistency of data in etcd. Without anything can be stored in a resource which can have security implications."